activerecord/lib/active_record/signed_id.rb

```diff
@@ -11,9 +11,10 @@ module SignedId
       # Set the secret used for the signed id verifier instance when using Active Record outside of \Rails.
       # Within \Rails, this is automatically set using the \Rails application key generator.
       class_attribute :signed_id_verifier_secret, instance_writer: false
+      ::ActiveRecord::Relation.include(SignedId::RelationMethods)
```

Comment:
Not sure if this is really the best way to do it. I would also explicitly include in in `relation.rb`?